app/clients/sms/aws_sns.py

```diff
@@ -66,12 +65,9 @@ def send_sms(self, to, content, reference, multi=True, sender=None):
             try:
                 start_time = monotonic()
                 response = client.publish(PhoneNumber=to, Message=content, MessageAttributes=attributes)
-            except botocore.exceptions.ClientError as e:
-                self.statsd_client.incr("clients.sns.error")
-                raise str(e)
             except Exception as e:
                 self.statsd_client.incr("clients.sns.error")
-                raise str(e)
```

Comment:
This stuff was redundant and also the raise didn't work! You can't raise a string, just an `Exception`.

Reply:
pylint for the win! I feel bad we had that for that long in the code. The whole error management does look weird in retrospective, thank you for refactoring.

Reply:
Ha, I didn't find it with pylint but by accidentally triggering the error locally (I was trying to send to a USA number without setting the `AWS_US_TOLL_FREE_NUMBER` variable)

Reply:
Okay. I thought it might be pylint because the latter did raise it as a potential problem when I looked at this code locally in vscode.